website/meta/universe.json

```diff
@@ -2762,7 +2762,35 @@
                 "github": "AMArostegui"
             },
             "category": ["nonpython"]
-        }
+        },
+	{
+	    "id": "ruts",
+	    "title": "ruTS",
+	    "slogan": "A library for statistics extraction from texts in Russian",
+	    "description": "The library allows extracting the following statistics from a text: basic statistics, readability metrics, lexical diversity metrics, morphological statistics",
+	    "github": "SergeyShk/ruTS",
+	    "pip": "ruts",
+	    "code_example": [
+		"import spacy",
+		"import ruts",
+		"",
+		"nlp = spacy.load('ru_core_news_sm')",
+		"nlp.add_pipe('basic', last=True)",
+		"doc = nlp('мама мыла раму')",
+		"doc._.basic.get_stats()"
+	    ],
+	    "code_language": "python",
+	    "url": "https://example.com",
```

Comment:
Do you want to add a specific link here? You already have the github link, so you could also just remove the "url" entry altogether.

Reply:
Done.